### ISSUE TITLE
Will this work when transcript has more text than the audio? 

### ISSUE THREAD

**Issue by vinaywadhwa** [nsheth12/canetis#7]
Hey there, 
I like your approach of improving over Gentle. I have a very specific problem i'm trying to solve. Maybe you can help? 
Say transcript has 100 lines but audio has only 20 lines of audio. Will Canetis find the 20 lines from the transcript and successfully transcribe just that portion of 20 lines (output file should have just those 20 lines, picked out from the transcript)? 
If not, how would you go about doing it? TIA :-)

**Comment by nsheth12** [comment]
The output will give every word in the transcript, along with information about if the word was found in the audio and if so, when in the audio it was found (see the "usage" section in the readme). So, it will be very easy to pick out the 20 lines worth of words that are in the audio.

TL;DR, Canetis should work for your use case. Give it a go and let us know if there are any issues!
